.github/workflows/test.yml

```diff
@@ -753,6 +755,7 @@ jobs:
       FF_TFM_PAYMENT_RECONCILIATION_ENABLED: 'true'
       FF_TFM_FACILITY_END_DATE_ENABLED: 'true'
       FF_TFM_DEAL_CANCELLATION_ENABLED: 'true'
+      # TODO DTFS2-6892: add FF_TFM_SSO_ENABLED feature flag here when e2e tests implimented
```

Comment:
Could/should we instead add this in now with a false value?

Reply:
No I dont think so -- this will be set to true eventually. Our FF helper defaults to false anyway.